src/hotspot/cpu/riscv/vm_version_riscv.cpp

```diff
@@ -99,6 +99,17 @@ void VM_Version::initialize() {
     }
   }
 
+  // Enable vendor specific features
```

Comment:
I think it's better to have vendor specific settings at the end of this function.